### PR TITLE
Develop.scale avatar

### PR DESCRIPTION
**Pull request by Vladlukhanin**
**Made/Proposed changes:**
- added ability for scaling the avatar.
- updated supported browsers in README.md

### CODE REVIEW

File: app/scripts/events.js [hunk 1]
```diff
@@ -426,32 +426,29 @@ define([
...
-                $chat.find('.pencil_active').removeClass('is-hidden');
-            });
-
-            $workspace.on('mouseleave', '.groupTitle .avatar', function() {
-                var $chat = $('.l-chat:visible');
-
-                $chat.find('.pencil_active').addClass('is-hidden');
-            });
+            $('body').on('click', '.j-changePic', function() {
+                var dialog_id = $(this).data('dialog');
```
Comment (dimaspirit) [inline app/scripts/events.js:430]: You need to make sure that `dialog_id` is not invalid.

File: app/scripts/events.js [hunk 2]
```diff
@@ -426,32 +426,29 @@ define([
...
                 var $chat = $('.l-chat:visible');
 
                 Dialog.changeAvatar($chat.data('dialog'), $(this), function(avatar) {
-                    if (!avatar) return false;
+                    if (!avatar) {
+                        return false;
+                    }
+
                     $chat.find('.avatar_chat').css('background-image', 'url(' + avatar + ')');
+                    $('.j-popupAvatar.is-overlay .j-avatarPic').attr('src', avatar);
```
Comment (dimaspirit) [inline app/scripts/events.js]: Wha are you need `.j-popupAvatar.is-overlay` ? 
.j-popupAvatar is not working?

File: app/scripts/helpers.js [hunk 3]
```diff
@@ -309,5 +309,25 @@ define([
         }
     };
 
+    Helpers.scaleAvatar = function($pic) {
+        var $chat = $pic.parents('.l-chat'),
+            name = $pic.data('name'),
+            url = $pic.css("background-image").replace(/.*\s?url\([\'\"]?/, '').replace(/[\'\"]?\).*/, ''),
```
Comment (dimaspirit) [inline app/scripts/helpers.js]: Use single quote

Comment (dimaspirit) [inline app/scripts/helpers.js]: What is this RegExp do? make comment in code!

File: app/scripts/views/dialog.js [hunk 4]
```diff
@@ -425,13 +425,14 @@ define([
                 html += '<div class="l-flexbox_inline">';
 
                 if (dialog.type === 3) {
-                    html += '<div class="contact-avatar avatar avatar_chat profileUserAvatar" style="background-image:url(' + icon + ')" data-id="' + user_id + '"></div>';
+                    html += '<div class="contact-avatar avatar avatar_chat profileUserAvatar scaleAvatar j-scaleAvatar" ';
```
Comment (dimaspirit) [inline app/scripts/views/dialog.js]: It's looks so sad, start using template engine

File: app/styles/modules/popup.scss [hunk 5]
```diff
@@ -260,11 +260,14 @@
     margin: 0;
     border: 0;
 }
-/* attach popup */
-.attach-photo {
+/* attach/avatar popup */
+.attach-photo, .avatar-pic {
```
Comment (dimaspirit) [inline app/styles/modules/popup.scss]: .attach-photo,
.avatar-pic {
...
}

Read one more time https://github.com/QuickBlox/CodeStyle-Guidelines-for-Developers/tree/master/web/css

File: app/scripts/events.js (hunk 1 above)
Comment (Vladlukhanin) [reply app/scripts/events.js:430]: I set _data-dialog_ attribute when building the UI for chat UI

File: app/index.html [hunk 6]
```diff
@@ -420,6 +420,14 @@ <h3 class="userDetails-filename profileUserName"></h3>
             </div>
         </section>
 
+        <section id="popupAvatar" class="popup popup_avatar j-popupAvatar">
+            <img class="avatar-pic j-avatarPic" src="" alt="avatar">
```
Comment (dimaspirit) [inline app/index.html]: don't write empty property 'src'

File: app/index.html [hunk 7]
```diff
@@ -420,6 +420,14 @@ <h3 class="userDetails-filename profileUserName"></h3>
...
+            <div class="avatar-description l-flexbox l-flexbox_flexbetween">
+                <span class="avatar-name j-avatarName"></span>
+                <a class="changePic j-changePic is-hidden">Change picture</a>
```
Comment (dimaspirit) [inline app/index.html]: Is it means 'avatar' ? Maybe  rename to 'Change your avatar' ?

File: app/index.html [hunk 8]
```diff
@@ -605,6 +613,76 @@ <h3 class="userDetails-filename profileUserName"></h3>
...
+                        <span class="pencil is-hidden"></span>
+                        <span class="triangle triangle_down"></span>
+                        <span class="triangle triangle_up is-hidden"></span>
+                    </div>
+                <% } %>
+            </div>
+            <div class="chat-controls">
+                <% if (type === 3) { %>
+                    <button class="btn_chat btn_chat_videocall videoCall">
+                        <img src="images/icon-videocall.svg" alt="videocall">
```
Comment (dimaspirit) [inline app/index.html]: http://prgssr.ru/development/alternativnyj-tekst-dlya-izobrazhenij.html#heading-section-3

alt="Make a Video Call"

File: app/index.html [hunk 9]
```diff
@@ -605,6 +613,76 @@ <h3 class="userDetails-filename profileUserName"></h3>
...
+                    </button>
+                    <button class="btn_chat btn_chat_audiocall audioCall">
+                        <img src="images/icon-audiocall.svg" alt="audiocall">
```
Comment (dimaspirit) [inline app/index.html]: Also 'Make a Audio Call'

File: app/index.html [hunk 10]
```diff
@@ -605,6 +613,76 @@ <h3 class="userDetails-filename profileUserName"></h3>
...
+                    </button>
+                    <button class="btn_chat btn_chat_profile userDetails" data-id="<%= user_id %>">
+                        <img src="images/icon-profile.svg" alt="profile">
+                    </button>
+                <% } %>
+                <button class="btn_chat btn_chat_add createGroupChat" data-ids="<%= occupantsIds.join() %>" data-private="1">
```
Comment (dimaspirit) [inline app/index.html:655]: Make a alt text is correct

**Pull request merged**